ls1intum/Artemis
# `Plagiarism checks`: Redesign user interface for plagiarism cases

### Checklist
#### General


#### Client

### Motivation and Context



The user interface for individual plagiarism cases (both for instructors and students) was a "working prototype" rather than a pleasing user experience.

### Description


I redesigned the client user interface for individual plagiarism cases, both the instructor and student view.
At the same time, I fixed the open issues #5537 and #5400.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Course (course name >50 characters)
- 1 Programming/Text/Modeling Exercise (exercise name >50 characters)
- Identical Submissions for the Exercise by each Student

1. Log in to Artemis
2. Navigate to Course Administration
3. Run the plagiarism detection for the exercise
4. Confirm the plagiarism to create a plagiarism case
5. Go to the course's plagiarism overview page
6. Click on a case (instructor view), notify the student (post length should be <1,000) and decide on a verdict (test any others parts of the user interface as well)
7. Login as the student
8. Go to the exercise and click on the "plagiarism case" button
9. Review the plagiarism case (student view)
10. Re-test these steps in any order and/or with other exercise types

### Review Progress





#### Code Review
#### Manual Tests

### Test Coverage


_unchanged_

### Screenshots




<img src="https://user-images.githubusercontent.com/102237363/183249045-75c3865d-dec3-4a83-9244-a5a2652ce791.png" width="400"><img src="https://user-images.githubusercontent.com/102237363/183249055-d23d78ed-428e-4365-ad8a-58a062f8b24c.png" width="400">

<img src="https://user-images.githubusercontent.com/102237363/183249067-3668afe6-e3aa-46ca-a984-ff3e548da6ca.png" width="400"><img src="https://user-images.githubusercontent.com/102237363/183249074-1688b28e-ac24-4b1c-8bf4-86042c1d97cd.png" width="400">
